packages/p2p/src/stores/floating-rate-store.js

```diff
@@ -0,0 +1,77 @@
+import { action, computed, observable } from 'mobx';
+import BaseStore from 'Stores/base_store';
+import { requestWS } from 'Utils/websocket';
+
+export default class FloatingRateStore extends BaseStore {
+    @observable fixed_rate_adverts_status;
+    @observable float_rate_adverts_status;
+    @observable float_rate_offset_limit;
+    @observable fixed_rate_adverts_end_date;
+    @observable exchange_rate;
+    @observable change_ad_alert;
+    @observable api_error_message = '';
+    @action.bound
+    setFixedRateAdvertStatus(fixed_rate_advert_status) {
+        this.fixed_rate_adverts_status = fixed_rate_advert_status;
+    }
+    @action.bound
+    setFloatingRateAdvertStatus(floating_rate_advert_status) {
+        this.floating_rate_advert_status = floating_rate_advert_status;
+    }
+    @action.bound
+    setFoatRateOffsetLimit(offset_limit) {
+        this.float_rate_offset_limit = offset_limit;
+    }
+    @action.bound
+    setFixedRateAdvertsEndDate(end_date) {
+        this.fixed_rate_adverts_end_date = end_date;
+    }
+    @action.bound
+    setChangeAdAlert(is_alert_set) {
+        this.change_ad_alert = is_alert_set;
+    }
+    @action.bound
+    setApiErrorMessage(api_error_message) {
+        this.api_error_message = api_error_message;
+    }
+    @action.bound
+    setP2PConfig() {
+        requestWS({ website_status: 1 }).then(response => {
+            if (!!response && response.error) {
+                this.setApiErrorMessage(response.error.message);
+            } else {
+                const { p2p_config } = response.website_status;
+                this.setFixedRateAdvertStatus(p2p_config.fixed_rate_adverts);
+                this.setFloatingRateAdvertStatus(p2p_config.float_rate_adverts);
+                this.setFoatRateOffsetLimit(p2p_config.float_rate_offset_limit);
+                this.setFixedRateAdvertsEndDate(p2p_config.fixed_rate_adverts_end_date || null);
+                this.setApiErrorMessage(null);
+            }
+        });
+    }
```

Comment:
Let's add this to the general store since this could be widely used across P2P.

Reply:
Yes, this function should definitely be in general-store.

Reply:
Renamed function as it was used to set only floating rate config values

Reply:
It's not about renaming the function, it's P2P general config and we should future proof it to get it from the general store and not from a specific one.

Reply:
Moving the function to general store